recipes/pyrfr/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyrfr" %}
+{% set version = "0.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 092cb7931486672ad00a71f37cc2e204cdb4b75ab0a47056667cc2683cc07dbd
+
+build:
+  number: 0
+  skip: True  # [py<30]
```

Comment:
Since CF doesn't build for py2 I don't think this is necessary. I'll let the CF experts weigh in on this though...

Reply:
No, it's not needed, I've just been adding too many `noarch` packages recently. 😄 I'll remove it.

Reply:
I'm just wondering if I should then add this to python itself, although the linter generally advises against it?

Reply:
IIUC CF builds for `>=py36`. If the package claims to work on any `py>=30` I probably wouldn't bother putting in a python constraint and just let the CF build config handle it.

I can't speak for CF though - I'm not sure of their official policy in this case...

Reply:
Yes you're 100% right, I was thinking of the `noarch` approach again. Your approach is correct.